sdcm/nemesis.py

```diff
@@ -358,6 +359,25 @@ def get_list_of_methods_by_flags(
         self.log.debug("Gathered subclass methods: {}".format(disrupt_methods_list))
         return disrupt_methods_list
 
+    def get_list_of_subclasses_by_property_name(self, list_of_properties_to_include):
+        flags = {flag_name: True for flag_name in list_of_properties_to_include}
+        subclasses_list = self._get_subclasses(**flags)
+        return subclasses_list
+
+    def get_list_of_disrupt_methods(self, subclasses_list):
+        disrupt_methods_list = []
+        for subclass in subclasses_list:
+            method_name = re.search(
+                r'self\.(?P<method_name>disrupt_[A-Za-z_]+?)\(.*\)', inspect.getsource(subclass), flags=re.MULTILINE)
+            if method_name:
+                disrupt_methods_list.append(method_name.group('method_name'))
+        self.log.debug("list of matching disrupions: {}".format(disrupt_methods_list))
```

Comment:
change to `self.log.debug("list of matching disrupions: %s", disrupt_methods_list)`

Reply:
Old style %-formatting isn't better or preferred over string formatting (if any, the old style should be removed).
I would have used f-format if not the old discussion and pylint idiotic errors.

Reply:
@roydahan you should do as Fabio suggested.
https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

Reply:
Not true anymore.
1. I don't get pylint errors for this.
2. Even If I had, I would change the pylintrc to "new" instead of "old" which doesn't have any real benefit.
3. Not only that, since pylint 2.4, we can also use fstrings if we want for logging and won't get this warning that doesn't matter in 99% of the times.

If you want to go into a debate, you're welcome to read issue 2395 in pylint.
(I don't want to link it so it won't link this PR there).